addons/notes/README.md

```diff
@@ -62,11 +62,11 @@ import someMarkdownText from './someMarkdownText.md';
 storiesOf('Component', module).add(
   'With Markdown',
   () => <Component />
-  { notes: someMarkdownText }
+  { notes: { markdown: someMarkdownText } }
 );
 ```
 
-If you want to use Github flavored markdown inline, use `notes: { markdownText: 'your md' }`:
+Similarly, if you want to use Github flavored markdown inline, use `notes: { markdownText: 'your md' }`:
```

Comment:
Looks like `markdownText` part is a typo in current docs, the example below uses `markdown`

Also, after your change there's no difference in handling inline vs imported markdown